core/nwb.base.yaml

```diff
@@ -44,29 +53,34 @@ datasets:
   - - null
     - null
     - 4
-  doc: An abstract data type for an image. Shape can be 2-D (x, y), or 3-D where the
+  doc: A type for storing image data directly. Shape can be 2-D (x, y), or 3-D where the
     third dimension can have three or four elements, e.g. (x, y, (r, g, b)) or
     (x, y, (r, g, b, a)).
   attributes:
   - name: resolution
     dtype: float32
     doc: Pixel resolution of the image, in pixels per centimeter.
     required: false
-  - name: description
+
+- neurodata_type_def: ExternalImage
+  neurodata_type_inc: BaseImage
+  doc: A type for referencing an external image file.
+  attributes:
+  - name: file_path
     dtype: text
-    doc: Description of the image.
-    required: false
+    doc: Path or URL to the external image file.
+    required: true
 
 - neurodata_type_def: ImageReferences
   neurodata_type_inc: NWBData
   dtype:
-    target_type: Image
+    target_type: BaseImage
     reftype: object
   dims:
   - num_images
   shape:
   - null
-  doc: Ordered dataset of references to Image objects.
+  doc: Ordered dataset of references to Image or ExternalImage objects.
```

Comment:
```suggestion
  doc: Ordered dataset of references to BaseImage (e.g., Image or ExternalImage) objects.
```

Reply:
Some extensions may want to extend `BaseImage`.

Reply:
Good point.